src/fides/api/api/v1/endpoints/config_endpoints.py

```diff
@@ -56,10 +57,17 @@ def update_settings(
     Only keys provided will be updated, others will be unaffected,
     i.e. true PATCH behavior.
     """
+
+    pruned_data = data.dict(exclude_none=True)
     logger.info("Updating application settings")
-    update_config: ApplicationConfig = ApplicationConfig.update_api_set(
-        db, data.dict(exclude_none=True)
-    )
+    update_config: ApplicationConfig = ApplicationConfig.update_api_set(db, pruned_data)
+
+    if (
+        "security" in pruned_data.keys()
+        and "cors_origins" in pruned_data.get("security").keys()
+    ):
+        load_updated_cors_domains(pruned_data["security"]["cors_origins"])
```

Comment:
That's great suggestion. So after this update `load_updated_cors_domains` will:

- accept a DB session and an app instance as parameters
- pull in the most up to date cors domains from `ConfigProxy` (which handles figuring out the config set and api set for us)
- update the current middleware domains based on what `ConfigProxy` returns

I'll update it to do this now

Reply:
This was accomplished with a slight modification here: https://github.com/ethyca/fides/pull/4073/commits/ccbbfbc157c6a15ce6a372f79a0d31984e6dc1fd

The `load_updated_cors_domains` was moved onto `ConfigProxy` since the two were couple together. Now the DB session doesn't need to be passed in.